### ISSUE TITLE
⌨ Redundant Links in the Tab order

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#8108]
## ⌨ Accessibility Issue

Having two “Manage all skills” links back to back in the tab order seemed unnecessary and was causing us both some confusion. 

### WCAG 2.1 Criteria

[WCAG 2.4.4 Link purpose (in context)](https://w3c.github.io/wcag/understanding/link-purpose-in-context)

## 🦋 Expected Behaviour

What should be happening?

## 🕵️ Details

Add any additional details that could assist with troubleshooting/fixing the issue.

- **Operating System**: Name and version (if possible).
- **Browser**: Name and version (if possible).
- **Assistive Technology**: Name and version (if possible).

## 📋 Steps to Reproduce

1. Go to the Skills showcase  using a screen reader (NVDA, Narrator, VoiceOver, Orca)
2. Tab through the links in the contents section "on this page" and into the main content links.
3. Listen to the link repetition before you can edit 

## 📸 Screenshot

[Skills Showcase image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBb0h4IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--1701787d3ee7367dd7c27bcf428fe5ce347e856e/image.png)

## 🙋‍♀️ Proposed Solution

[H2: Combining adjacent image and text links for the same resource](https://w3c.github.io/wcag/techniques/html/H2) 

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by substrae** [comment]
I think this might have been resolved by the changes in https://github.com/GCTC-NTGC/gc-digital-talent/issues/8103

**Comment by RM-1978** [comment]
@substrae Happy to add the acceptance criteria to #8103

**Comment by petertgiles** [comment]
Hi @RM-1978.  You would like the fix for this specific instance to be part of #8103 and the scope for this issue to be site-wide?

**Comment by RM-1978** [comment]
Hi @petertgiles, thank you for your question. That would be cool if the fix for this specific instance to be part of #8103, but yes, it would be nice if any instances of Redundant links across the site were checked to see if we need to consider this fix or the WCAG suggested fix [H2: Combining adjacent image and text links for the same resource](https://w3c.github.io/wcag/techniques/html/H2) would apply.
